app/guided.py

```diff
@@ -171,6 +170,33 @@ async def execute_subop(self, ability_ids: List[str], agent: Agent = None, goals
 
     """ PRIVATE """
 
+    async def _show_attack_graph(self, g):
+        """DEBUG function -
```

Comment:
should this be limited to only running if `--log DEBUG` is used?

Reply:
Is the log level directly accessible anywhere? Just looking through `server.py` it seems like it's used to configure `logging` and then gets dropped. `_show_attack_graph` is currently set to only run based on a separate debug boolean in the planner yaml, which seemed sufficient.